frontend/lib/src/components/elements/Markdown/Markdown.test.tsx

```diff
@@ -40,10 +38,10 @@ const getProps = (
 describe("Markdown element", () => {
   it("renders markdown as expected", () => {
     const props = getProps()
-    const wrap = shallow(<Markdown {...props} />)
-    const elem = wrap.get(0)
-    expect(elem.props.className.includes("stMarkdown")).toBeTruthy()
-    expect(elem.props.style.width).toBe(100)
+    render(<Markdown {...props} />)
+    const markdown = screen.getByTestId("stMarkdown")
+    expect(markdown).toBeInTheDocument()
+    expect(getComputedStyle(markdown).width).toBe("100px")
   })
   /* MAYBE ADD TESTS?
```

Comment:
We should probably take this opportunity to add these tests & remove this TODO -- Sorry 😅

Reply:
I just looked at `StreamlitMarkdown.test.tsx` and that looks like it uses the `allowHTML` prop and different types of markdowns so I think we can just remove this comment as `Markdown` just uses `StreamlitMarkdown`.